src/eng_front.c

```diff
@@ -263,6 +263,7 @@ static int bind_helper(ENGINE *e)
 			!ENGINE_set_load_privkey_function(e, load_privkey)) {
 		return 0;
 	} else {
+		ENGINE_set_table_flags(ENGINE_TABLE_FLAG_NOINIT);
```

Comment:
We might want to set this in engine_unlocked_init() or make engine_unlocked_init() re-entrant.

Reply:
I'd much like to see `engine_unlocked_init()` re-entrant. Can you do that? How quickly?

Reply:
Oh, and why is Travis complaining again?

Reply:
Ouch... I just realized that `engine_unlocked_init()` is in `openssl/crypto/engine/eng_init.c` file. Therefore it would require OpenSSL team to agree to merge the fix - assuming the fix does get created...

Reply:
@mouse07410 sorry I meant to say make ctx_init() re-entrant.

Reply:
> Didn't work

You mean - you tried to make `ctx_init()` re-entrant, and the attempt failed?

Reply:
Yep